pkg/doc/signature/proof/utils.go

```diff
@@ -20,9 +20,14 @@ func GetProofs(jsonLdObject map[string]interface{}) ([]*Proof, error) {
 		return nil, ErrProofNotFound
 	}
 
-	typedEntry, ok := entry.([]interface{})
-	if !ok {
-		return nil, errors.New("expecting []interface{}, got something else")
+	var typedEntry []interface{}
+	switch te := entry.(type) {
+	case []interface{}:
+		typedEntry = te
+	case map[string]interface{}:
+		typedEntry = []interface{}{te}
+	default:
+		return nil, errors.New("expecting []interface{} or map[string]interface{}, got something else")
```

Comment:
can be more succinct :). don't need ", got something else".

Reply:
Agree :) I took this pattern from @sandrask's  `signature` package - one of the cleanest and clearest packages i've seen 👍